pkg/authorization/apiserver/admission/restrictusers/restrictusers_test.go

```diff
@@ -386,7 +386,7 @@ func TestAdmission(t *testing.T) {
 			&user.DefaultInfo{},
 		)
 
-		err = plugin.(admission.ValidationInterface).Validate(attributes)
+		err = plugin.(admission.ValidationInterface).Validate(attributes, nil)
```

Comment:
How much do I hate that this change was merged, knowing this will follow in origin.